public/public.php

```diff
@@ -472,12 +472,12 @@ public function list_post_attachments_with_sources( $post_id = 0 ) {
 			ISC_Log::log( sprintf( 'going through %d attachments', count( $attachments ) ) );
 			$atts = [];
 			foreach ( $attachments as $attachment_id => $attachment_array ) {
-				$use_standard_source = Standard_Source::use_standard_source( $attachment_id );
-				$source              = self::get_image_source_text( $attachment_id );
+				$image_uses_standard_source = Standard_Source::use_standard_source( $attachment_id );
+				$source                     = self::get_image_source_text( $attachment_id );
 
 				// check if source of own images can be displayed
-				if ( ( $use_standard_source === '' && $source === '' ) || ( $use_standard_source !== '' && $exclude_standard ) ) {
-					if ( $use_standard_source !== '' && $exclude_standard ) {
+				if ( ( ! $image_uses_standard_source && $source === '' ) || ( $image_uses_standard_source && $exclude_standard ) ) {
+					if ( $image_uses_standard_source && $exclude_standard ) {
 						ISC_Log::log( sprintf( 'image %d: "own" sources are excluded', $attachment_id ) );
 					} else {
 						ISC_Log::log( sprintf( 'image %d: skipped because of empty source', $attachment_id ) );
```

Comment:
**Suggestion:** The condition in line 479 seems to be a bit complex and could be simplified for better readability. You could separate the conditions into two if statements. [maintainability]
```suggestion
    if ( ! $image_uses_standard_source && $source === '' ) {
      ISC_Log::log( sprintf( 'image %d: skipped because of empty source', $attachment_id ) );
    } 
    if ( $image_uses_standard_source && $exclude_standard ) {
      ISC_Log::log( sprintf( 'image %d: "own" sources are excluded', $attachment_id ) );
    }
```